### DISCUSSION TITLE
Feature Proposal - Hover-Text for Approximate Title Detection

### DISCUSSION THREAD

**Discussion by vibe-code-repo** [icereed/paperless-gpt#9] [Ideas]
Would it be possible to display the document's content as a hover text, which shows when the title cannot be recognized correctly? This way, one can read in the text what a correct title would be and possibly derive a suitable tag from it.

**Comment by icereed** [comment]
Cool idea, do you think more of a text only thing or along with a thumbnail?

**Comment by icereed** [comment]
This feature is implemented here #11

**Reply by icereed** [reply]
Feature is implemented:
<img width="1063" alt="image" src="https://github.com/user-attachments/assets/8386d9e8-edff-430f-b2e7-486c09f61662">
